pkg/controller.v1/tensorflow/util.go

```diff
@@ -41,12 +36,78 @@ func GetPortFromTFJob(tfJob *tfv1.TFJob, rtype commonv1.ReplicaType) (int32, err
 	return tfv1.DefaultPort, nil
 }
 
-// ContainChieforMasterSpec returns true if the tfjob contains chief or master spec.
-func ContainChieforMasterSpec(replicas map[commonv1.ReplicaType]*commonv1.ReplicaSpec) bool {
+// ContainsChiefOrMasterSpec returns true if the tfjob contains chief or master spec.
+func ContainsChiefOrMasterSpec(replicas map[commonv1.ReplicaType]*commonv1.ReplicaSpec) bool {
 	if _, ok := replicas[tfv1.TFReplicaTypeChief]; ok {
 		return true
 	} else if _, ok := replicas[tfv1.TFReplicaTypeMaster]; ok {
 		return true
 	}
 	return false
 }
+
+func getContainerExitCode(pod *corev1.Pod) int32 {
```

Comment:
Can you add some comments where're auxiliary methods from? like which delete filed.

Reply:
sure